cremalab/sarah-todo
# Adasp 28 edit task

## Description

DESCRIPTION_HERE

### 🔗 Reference

> _Relevant links (e.g. issue, design, etc.)_

- [ADASP-28](https://crema.atlassian.net/jira/software/projects/ADASP/boards/108?selectedIssue=ADASP-28)

### 📋 Acceptance Criteria

> _Checked off by PR **Assignees**_

- [ ] “Edit Task” title
- [ ] delete icon next to "Edit Task"
- [ ] cancel and save buttons
- [ ] input is pre-filled with current description
- [ ] clicking trashcan icon deletes the task
- [ ] changing text of task and clicking Save button changes the text of the task





### 👩‍🔬 Test Instructions

> _Provided by PR **Assignees**_

1. Create two new tasks by clicking the "Create New Task" button and adding two separate tasks.
2. Click on the first task card and click the trash can. The task should be deleted.
3. Click the remaining task card and edit the text in the input.
4. Text on the card should now be the new text you entered.

## 🔎 Reviewer Checklist

> _Checked off by PR **Reviewers**_




---

> ### Roles & Responsibilities
>
> #### 👨‍💻 Assignee
>
> - Initiator of this PR (be sure to set in GitHub UI)
> - Addresses feedback and change requests
> - Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
> #### 👩‍💻 Reviewer
>
> - Invited to review PR by **Assignee** (via GitHub UI)
> - Is expected to complete a review and address followup
